libs/community/langchain_community/document_loaders/notiondb.py

```diff
@@ -20,13 +20,17 @@ class NotionDBLoader(BaseLoader):
         database_id (str): Notion database id.
         request_timeout_sec (int): Timeout for Notion requests in seconds.
             Defaults to 10.
+        filter_object (Dict[str, Any]): Filter object used to limit returned
```

Comment:
Could you provide an example of how to use the filter object? (This is probably the most important thing  otherwise it's not really obvious how to use this functionality)

Is there also a URL to notion db API that could be included so folks could see the documentation?